index.html

```diff
@@ -17,5 +17,8 @@
     font-weight: 100;
     line-height: 1em;
 ">Let's go fishing!</h1>
+<audio controls="controls" style="width:0;height:0;" autoplay>
+  <source src="best-song-ever.mp3" type="audio/mp3">
```

Comment:
You'll have to host the song elsewhere and provide the full url to it here. It only pulls through the contents of this file, no others.